libr/anal/bb.c

```diff
@@ -173,6 +174,7 @@ R_API void r_anal_bb_set_offset(RAnalBlock *bb, int i, ut16 v) {
 		if (i >= bb->n_op_pos) {
 			bb->n_op_pos = i * 2;
 			bb->op_pos = realloc (bb->op_pos, bb->n_op_pos * sizeof (*bb->op_pos));
```

Comment:
realloc must go into a temporal vairable to be able to check if new reallocated buffer is not null, otherwise it will be leaking or null dereferecing